urumanifest/commit.py

```diff
@@ -46,8 +55,14 @@ def _hash_asset(args: Tuple[Path, Path]) -> Tuple[Path, str, int]:
     server_path, source_path = args
     # One day, we will not use such a vulnerable hashing algo...
     h = hashlib.md5()
-    with source_path.open("rb") as in_stream:
-        _io_loop(in_stream, h.update)
+    if source_path.is_dir:
```

Comment:
```suggestion
    if source_path.is_dir():
```

This would previously always evaluate to truthiness because you were testing the bound method instead of calling it.